config/settings.yml

```diff
@@ -182,3 +182,5 @@ smtp_password: null
 #  -----BEGIN PRIVATE KEY-----
 #  ...
 #  -----END PRIVATE KEY-----
+# Override Firefox binary used in system tests
+#system_test_firefox_binary:
```

Comment:
Can we put this in `config/settings/test.yml` as it's only for testing?

Reply:
Is teaspoon supposed to be run in test environment?

Reply:
That's actually an excellent question - my answer would be yes, and it doesn't work in my dev environment unless I force it to, but currently there is nothing to make that happen by default.

Reply:
Actually the github workflow does run it with `RAILS_ENV=test` set.

Reply:
Moved to `config/settings/test.yml`.

Then we have to update `CONTRIBUTING.md` to explain how to run teaspoon.